wsic/utils.py

```diff
@@ -13,46 +13,31 @@ def downsample_shape(
     downsample: int,
     rounding_func: Callable[[Number], int] = floor,
 ) -> Tuple[int, ...]:
-    r"""Calculate the shape of an array after downsampling by a factor.
-
+    """Calculate the shape of an array after downsampling by a factor.
     The shape is calculated by dividing the shape of the baseline array
     by the downsample factor. The output is rounded to the nearest
     integer using the provided rounding function. E.g. for a founding
     function of `floor`, the following opertion is performed
     :math:`\lfloor \frac{shape}{downsample} \rfloor`. If a channel
     dimension is specified, the dimension is left unchanged.
-
     Args:
         baseline_shape (Tuple[int, ...]):
             The shape of the array to downsample.
         downsample (int):
-            The downsample factor.
+            The downsample factor. Must be greater than 0.
         rounding_func (Callable[[int], int]):
             The rounding function to use. Defaults to floor. Any
             function which takes a single Number and returns an int such
             as `math.floor` or `math.ceil` can be used. Note that the
             behaviour of floor differs for negative numbers, e.g.
             floor(-1) = -2. The `int` function is sigificantly faster
             than floor.
-
     Returns:
         Tuple[int, ...]:
             The shape of the downsampled array.
-
-    Examples:
-        >>> dowmsample_shape((13, 13), 2)
-        (6, 6)
-
-        >>> dowmsample_shape((13, 13, 3), 2)
-        (6, 6, 3)
-
-        >>> dowmsample_shape((13, 13, 3), 2, channel_dim=2)
-        (6, 6, 3)
-
-        >>> downsample_shape((13, 13, 3), 2, -1, ceil)
-        (7, 7, 3)
-
```

Comment:
Is there a reason for removing all of the examples? Please leave examples in the documentation. I see there is a typo here and that 'downsample' should be 'downsample'.